notebooks/official/CODEOWNERS

```diff
@@ -4,8 +4,8 @@
 # review when someone opens a pull request.
 * @GoogleCloudPlatform/vertex-ai-samples-contributors @GoogleCloudPlatform/caiis-tw
 
-# matching_engine folder
-/matching_engine @shenzhimo2 @ivanmkc
+# vector_search folder
+/vector_search @shenzhimo2 @ivanmkc
```

Comment:
missed corrections in the following PR (maybe)
- https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/3096